internal/services/v0/namespace_test.go

```diff
@@ -111,29 +90,30 @@ func TestNamespaceChanged(t *testing.T) {
 			),
 			"cannot delete relation `viewer` in definition `folder`, as a relationship exists under it",
 		},
-		{
-			"relation referenced by tuples",
-			ns.Namespace(
-				"folder",
-				ns.Relation("anotherrel",
-					nil,
-					ns.AllowedRelation("folder", "..."),
-				),
-				ns.Relation("viewer",
-					nil,
-					ns.AllowedRelation("user", "..."),
-				),
-			),
-			[]*v0.RelationTuple{tuple.MustParse("folder:somefolder#anotherrel@folder:somefolder#viewer")},
-			ns.Namespace(
-				"folder",
-				ns.Relation("anotherrel",
-					nil,
-					ns.AllowedRelation("folder", "..."),
-				),
-			),
-			"cannot delete relation `viewer` in definition `folder`, as a relationship references it",
-		},
+		// TODO: review this test
```

Comment:
Review it for what? Was it reviewed? Is it just failing?

Reply:
Fixed - the tuple being written wasn't valid under the namespace config; the previous test wasn't type-checking